node/rpc/src/chain_api.rs

```diff
@@ -0,0 +1,92 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+use crate::State;
+use blocks::BlockHeader;
+use blockstore::BlockStore;
+use cid::json::CidJson;
+use jsonrpc_v2::{Data, Error as JsonRpcError, Params};
+use message::{
+    signed_message::json::SignedMessageJson, unsigned_message::json::UnsignedMessageJson,
+    UnsignedMessage,
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize)]
+pub(crate) struct BlockMessage {
+    #[serde(rename = "BlsMessages")]
+    pub bls_msg: Option<Vec<UnsignedMessageJson>>,
+    #[serde(rename = "SecpkMessages")]
+    pub secp_msg: Option<Vec<SignedMessageJson>>,
+    #[serde(rename = "Cids")]
+    pub cids: Option<Vec<CidJson>>,
+}
```

Comment:
so since there are a lot of cloning to transform these into the wrapper types, probably best to tag each to not require changing them, unfortunately serde does not have an ergonomic way of doing this https://github.com/serde-rs/serde/issues/723 so lets chat about this later.

Reply:
Will need for #470 to be merged.